### ISSUE TITLE
View queue, topic and subscription message counts (including scheduled)

### ISSUE THREAD

**Issue by alexespencer** [Azure/azure-sdk-for-python#20245]
I would like to be able to obtain the active, pending and dead-letter message counts from a queue, topic or subscription

Either via ServiceBusAdministrationClient or the receiver itself?

You used to be able to do this with the old library?

**Comment by swathipil** [comment]
@alexespencer Thanks for your question!

The active, scheduled, and dead-letter message counts are all provided by the `*RuntimeProperties` objects returned by the `list_*_runtime_properties` methods on the `ServiceBusAdministrationClient`. 

For example, you can use the [`get_queue_runtime_properties`](https://docs.microsoft.com/en-us/python/api/azure-servicebus/azure.servicebus.management.servicebusadministrationclient?view=azure-python#get-queue-runtime-properties-queue-name----kwargs-)  method to obtain the [QueueRuntimeProperties](https://docs.microsoft.com/en-us/python/api/azure-servicebus/azure.servicebus.management.queueruntimeproperties?view=azure-python) object:
```
import os
import uuid
from azure.servicebus.management import ServiceBusAdministrationClient

CONNECTION_STR = os.environ['SERVICE_BUS_CONNECTION_STR']
QUEUE_NAME = os.environ['SERVICE_BUS_QUEUE_NAME']

servicebus_mgmt_client = ServiceBusAdministrationClient.from_connection_string(CONNECTION_STR)

# below returned object is a QueueRuntimeProperties object
queue_info = servicebus_mgmt_client.get_queue_runtime_properties(QUEUE_NAME)

# access the active, scheduled, and dead-letter message counts
print(queue_info.active_message_count)
print(queue_info.scheduled_message_count)
print(queue_info.dead_letter_message_count)
```

You can also find similar methods for subscription and topic [here](https://docs.microsoft.com/en-us/python/api/azure-servicebus/azure.servicebus.management.servicebusadministrationclient?view=azure-python#get-subscription-runtime-properties-topic-name--subscription-name----kwargs-) and [here](https://docs.microsoft.com/en-us/python/api/azure-servicebus/azure.servicebus.management.servicebusadministrationclient?view=azure-python#get-topic-runtime-properties-topic-name----kwargs-).

Thanks!

**Comment by swathipil** [comment]
@alexespencer - You mentioned that you were able to find the active, scheduled, and dead-letter message counts in the "old library". Out of curiosity, were you referring to `azure-servicebus==0.50.3` or a different version? This would be useful to know so that we can update our [migration guide](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/servicebus/azure-servicebus/migration_guide.md#working-with-administration-client) with this info to help others that have the same question as you. Thanks!

**Comment by alexespencer** [comment]
Yes, that is correct. I also checked through the examples you have in the
git repo (like receiving / sending etc) and there isn't an example that
does this. So that could help too?

**Comment by swathipil** [comment]
Very helpful to know, thanks!
